src/sentry/features/__init__.py

```diff
@@ -212,6 +212,7 @@
 default_manager.add("organizations:dynamic-sampling", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:dynamic-sampling-transaction-name-priority", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:escalating-issues", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+default_manager.add("organizations:escalating-issues-ui", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, FeatureHandlerStrategy.REMOTE)
```

Comment:
What's the difference between the one just above and this one?

Reply:
`-ui` and its flagr enabled. i can put a better comment